### PR TITLE
Add Neural Network Libraries

### PR DESCRIPTION
**Pull request by TE-YasunariHashimoto**
If you're adding a project to the Python 3 statement, please ensure you have:

- [x] Discussed dropping Python 2 support with all maintainers
- [x] Discussed signing the statement with all maintainers
- [x] Added a suitable logo if the project has one
  - [x] Approximately square logos generally fit better than banners
  - [ ] Compressed the logo to save bandwidth (run `scripts/squash-images.sh` on it)
    - [x] Add a link to the logo instead of uploading it.
- [x] Checked the Netlify preview made from the pull request

I added a link to the logo instead of uploading it.  I am one of the main maintainers of the Neural Network Libraries.  You can see my account the following https://github.com/sony/nnabla/graphs/contributors .

### CODE REVIEW

File: _sections/30-projects.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ These projects pledge to drop Python 2 support in or before 2020.
 - [![](assets/scikit-image.png)Scikit Image](http://scikit-image.org/) <!-- url:https://github.com/scikit-image/scikit-image sg:3069 -->
 - [![](assets/sphinx-doc.png)Sphinx](https://www.sphinx-doc.org/) <!-- url:https://github.com/sphinx-doc/sphinx sg:2681 -->
 - [![](assets/geopy.png)geopy](https://geopy.readthedocs.io/) <!-- url:https://github.com/geopy/geopy sg:2476 -->
+- [![](https://nnabla.org/logos/logo3.png)Neural Network Libraries](https://nnabla.org/)
```
Comment (takluyver) [inline _sections/30-projects.md]: Could you put an HTML comment with the Github URL, like the other lines? There's a script that gets the updated star counts using these and resorts the list.

**Comment by takluyver** [comment]
This LGTM apart from that comment; thanks.

File: _sections/30-projects.md (hunk 1 above)
Comment (TE-YasunariHashimoto) [reply _sections/30-projects.md]: Sure, got it.  Thank you for the feedback.

Comment (TE-YasunariHashimoto) [reply _sections/30-projects.md]: I added GitHub URL.  Please confirm the change thanks.

**Comment by takluyver** [comment]
Thanks!

**Pull request merged**